health-reform-observer.csl

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Health Reform Observer</title>
```

Comment:
It looks like the journal's full title is "Health Reform Observer – Observatoire des Réformes de Santé", so it might be better to change the title to `<title>Health Reform Observer - Observatoire des Réformes de Santé</title>` (we prefer regular hyphens over en-dashes) and the file name to "health-reform-observer-observatoire-des-reformes-de-sante.csl" (with matching style ID and "self" link). Agreed?